parts/k8s/kubeproxystart.ps1

```diff
@@ -16,6 +17,12 @@ while (!$hnsNetwork) {
     $hnsNetwork = Get-HnsNetwork | ? Name -EQ $KubeNetwork
 }
 
+# add dualstack feature gate if dualstack enabled
+$isDualStackEnabled = ("--feature-gates=IPv6DualStack=true" | ? { $Global:ClusterConfiguration.Kubernetes.Kubelet.ConfigArgs -match $_ }) -ne $null
```

Comment:
Would this check fail if `IPv6DualStack` feature gate was not first in the list `--feature-gates=CSIInlineVolume=true,IPv6DualStack=true`?

Reply:
you're right, it will fail. let me change the match to "IPv6DualStack=true"